### PR TITLE
Update es.yml

### PR DESCRIPTION
**Pull request by 103percent**
More Spanish strings

### CODE REVIEW

**Review by KimberleyCook — CHANGES_REQUESTED** [review]
Hey, all looks good. Except codebar is always a lowercase, so always just `codebar`. Could you amend please, once done I'll approve and merge. 

Thaaaanks :)

**Comment by 103percent** [comment]
So the Codebar/CodeBar instances should all be fixed at this point

File: config/locales/es.yml [hunk 1]
```diff
@@ -223,7 +223,7 @@ es:
         intro: "Si desea asistir a un estudiante, asegúrese de conocer a nuestro"
         criteria_link: "criterio de elegibilidad"
         criteria_brief: "como nuestros talleres solo están disponibles para las mujeres, LGBTQ, personas de grupos minoritarios subrepresentados."
-        github: "Por último, cuando hace clic en el botón Registrarse, se lo llevará a algo llamado GitHub. Después de iniciar sesión con una cuenta GitHub, volverá al sitio web de CodeBar. Si aún no tiene una cuenta, Tendrás que crear uno. ¡No tarda mucho tiempo y es algo que vendrá de manera útil cuando comience con la codificación!"
+        github: "Por último, cuando hace clic en el botón Registrarse, se lo llevará a algo llamado GitHub. Después de iniciar sesión con una cuenta GitHub, volverá al sitio web de Codebar. Si aún no tiene una cuenta, Tendrás que crear uno. ¡No tarda mucho tiempo y es algo que vendrá de manera útil cuando comience con la codificación!"
```
Comment (KimberleyCook) [inline config/locales/es.yml:226]: There is one instance here where codebar is still uppercase.

**Comment by KimberleyCook** [comment]
@103percent thanks for updating, there is one place it is still uppercase. Once amended I'll approve.

And thank you so much for doing all these translations, it is hugely appreciated :D

File: config/locales/es.yml (hunk 1 above)
Comment (103percent) [reply config/locales/es.yml]: Ah yes! I'll just change that

**Comment by 103percent** [comment]
> @103percent thanks for updating, there is one place it is still uppercase. Once amended I'll approve.
> 
> And thank you so much for doing all these translations, it is hugely appreciated :D

No problem! I do some when I can. I have noticed that some new content has been added, including a lot of legalese. I was wondering what the implications were for translating legal language and what I should do about it?

**Review by KimberleyCook — APPROVED** [review]
Perfect, thanks @103percent 🎉

**Pull request merged**